src/frontend/src/components/SEO.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Helmet from 'react-helmet';
+import useSiteMetadata from '../hooks/use-site-metadata';
+
+function SEO({ description_, lang, meta, title_ }) {
+  const { title, description } = useSiteMetadata();
+  const metaDescription = description_ || { description };
+  console.log(title);
+  console.log(title_);
```

Comment:
These are here for testing I guess?  Remember to remove.

Reply:
Removed